draft-ietf-rats-eat.md

```diff
@@ -1862,12 +1858,12 @@ the EAT they are consuming.
 ## Detached EAT Bundle Security Considerations
```

Comment:
It's actually detached digest considerations. A DEB is only one option for detached digests.

Reply:
Please see latest change